backend/src/database/daos/users/user.dao.service.ts

```diff
@@ -43,19 +31,7 @@ export class UserDaoService {
       return existingEntry;
     }
 
-    return await this.userModel.query().insert(user).returning('*');
-  }
-
-  deleteUserAccount(userId: string, trx: Transaction) {
-    return this.userModel.query(trx).deleteById(userId);
-  }
-
-  async checkUsernameExists(username: string) {
-    return !!(await this.userModel.query().findOne({ username }));
-  }
-
-  async checkTelegramHandleExists(telegramHandle: string) {
-    return !!(await this.userModel.query().findOne({ telegramHandle }));
+    return await this.userModel.query().insertGraph(user);
```

Comment:
Curious, what is this insertGraph thing?

Reply:
abit like mongo insert. lets us specify a nested object, and the ORM will jus create the entries in the related tables for us